src/styles/changes.scss

```diff
@@ -0,0 +1,11 @@
+.changes-page {
+    .content h2.date-and-month {
+        margin-top: 2em;
+        padding-bottom: 16px;
+        border-bottom: 1px solid lightgray;
+    }
+
+    .hero-body {
+        max-width: 720px;
```

Comment:
Hey @toolness! Check out the .content-wrapper .tight class defined in [_custom.scss](https://github.com/JustFixNYC/justfix-website/blob/master/src/styles/_custom.scss). Would suggest reusing this class instead of adding this custom CSS here but up to you!

Reply:
Oh good idea, thanks!  I just fixed this in 0fda6136.